### ISSUE TITLE
Update guide links referring to other guides to use the new subdomain version (post-migration)

### ISSUE THREAD

**Issue by christophermather** [18F/guides#343]
### A description of the work
There are places in our new guides that still explicitly point to the old URLs for the guides (things like `accessibility.18f.gov` when it should be `guides.18f.gov/accessibility/`). While those old URLs should be set up to redirect -- done as a part of each individual site migration -- we will want to use the new canonical forms.

### Point of contact on this issue

@eric-gade 

### Reproduction steps (if necessary)

- Replace all references to URLs that include `<guide-name>.18f.gov` to be `guides.18f.gov/<guide-name>`
  
I can't be sure, but something like the following snippet might be useful:
```shell
grep -rl "accessibility.18f.gov" . | xargs sed -i '' -e 's/accessibility\.18f\.gov/guides\.18f\.gov\/accessibility/g'
```

### Skills Needed

- [ ] Any Human
- [ ] Design
- [ ] Content
- [ ] Engineering
- [ ] Acquisition
- [ ] Product
- [ ] Other

### Does this need to happen in the next 2 weeks?

- [ ] Yes
- [ ] No

### How much time do you anticipate this work taking?

Shouldn't take more than an hour or two

### Acceptance Criteria

- [ ] There are no references or links to any guides in the codebase that are in the `<guide-name>.18f.gov`
